docs/ibc/light-clients/overview.md

```diff
@@ -1,3 +1,45 @@
 <!--
 order: 1
--->
\ No newline at end of file
+-->
+
+# Overview 
+
+Learn how to build IBC lightclient modules and fulfill the interfaces required to integrate with core IBC. {synopsis}
+
+The following aims to provide a high level IBC lightclient module developer guide. The lifecycle of IBC lightclients are created and maintained by the core IBC `MsgServer` and the abstractions provided by the `02-client` submodule. A lightclient module developer is only required to implement the methods and fulfil the expectations of a defined set of interfaces. In ibc-go these interfaces reside within the `core/modules/exported` package. 
```

Comment:
I'm a bit confused by the terminology "The lifecycle of IBC light clients". To me, "lifecycle" implies a beginning and end (packets have a clear beginning and end), but light clients aren't meant to have an end. Is there different wording we could use?

Reply:
Yeah good point. I'll use your suggestion in the other comment! 👍